src/core/game/GameImpl.ts

```diff
@@ -103,11 +103,11 @@ export class GameImpl implements Game {
       const numPlayerTeams = _config.numPlayerTeams();
       if (numPlayerTeams < 2)
         throw new Error(`Too few teams: ${numPlayerTeams}`);
-      if (numPlayerTeams >= 3) this.playerTeams.push(Team.Teal);
-      if (numPlayerTeams >= 4) this.playerTeams.push(Team.Purple);
-      if (numPlayerTeams >= 5) this.playerTeams.push(Team.Yellow);
+      if (numPlayerTeams >= 3) this.playerTeams.push(Team.Yellow);
+      if (numPlayerTeams >= 4) this.playerTeams.push(Team.Green);
+      if (numPlayerTeams >= 5) this.playerTeams.push(Team.Teal);
       if (numPlayerTeams >= 6) this.playerTeams.push(Team.Orange);
-      if (numPlayerTeams >= 7) this.playerTeams.push(Team.Green);
+      if (numPlayerTeams >= 7) this.playerTeams.push(Team.Purple);
```

Comment:
Recommendation from @mOctave 

```suggestion
      if (numPlayerTeams >= 5) this.playerTeams.push(Team.Purple);
      if (numPlayerTeams >= 6) this.playerTeams.push(Team.Orange);
      if (numPlayerTeams >= 7) this.playerTeams.push(Team.Teal);
```